### PR TITLE
Net 6555 Adds field to MeshGW Crd based on new Consul Proto

### PR DESCRIPTION
**Pull request by missylbytes**
Changes proposed in this PR:
- Updates the MeshGW CRD with new Consul Proto
- Fixes Makefile command that was broken because of spacing and bash.

How I've tested this PR:
- 👀 for MeshGW CRD
- Ran `ensure-controller-gen-version` to make sure it wouldn't fail.
How I expect reviewers to test this PR:
- 👀 for MeshGW CRD
- Run `ensure-controller-gen-version` to make sure it wouldn't fail.

### CODE REVIEW

File: charts/consul/true.yml
Comment (nathancoleman) [inline charts/consul/true.yml:1]: Why is this file added? `true.yaml` seems like an odd name 😕

Comment (missylbytes) [reply charts/consul/true.yml:1]: It isn't supposed to be here.

**Review by nathancoleman — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -232,13 +232,13 @@ ensure-controller-gen-version: ## Ensure controller-gen version is v0.12.1.
...
 	@echo "You don't have $(CONTROLLER_GEN), please install it first."
 else
-	ifeq (, $(shell $(CONTROLLER_GEN) --version | grep v0.12.1))
-		@echo "controller-gen version is not v0.12.1, uninstall the binary and install the correct version with 'make get-controller-gen'."
-		@echo "Found version: $(shell $(CONTROLLER_GEN) --version)"
-		@exit 1
-	else
-		@echo "Found correct version: $(shell $(CONTROLLER_GEN) --version)"
-	endif
+ifeq (, $(shell $(CONTROLLER_GEN) --version | grep v0.12.1))
```
Comment (sarahalsmiller) [inline Makefile:235]: Thank you 🚀

**Review by sarahalsmiller — APPROVED** [review]
